person_capture/gui_app.py

```
@@ -2030,25 +2028,29 @@ def _compose_dataset_crop(
-                return list(validated_user_ratios)
+            allow_landscape = profile == "body"
+            seed = list(validated_user_ratios) + preferred
             out: list[str] = []
-            for rs in preferred:
+            for rs in seed:
                 try:
-                    parse_ratio(rs)
+                    rw, rh = parse_ratio(rs)
+                    aspect = float(rw) / max(1e-6, float(rh))
                 except Exception:
                     continue
```

Ruff (0.15.11)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)